resources/kcp/charts/kyma-metrics-collector/prometheus/alerts.yaml

```diff
@@ -64,4 +64,13 @@ groups:
       labels:
         severity: critical
       annotations:
-        description: Rate of increase in errors for the requests from KMC to Gardener.
\ No newline at end of file
+        description: Rate of increase in errors for the requests from KMC to Gardener.
+  - name: kmc.rules.processing
+    rules:
+    - alert: KMCProcessingClustersFromKEBFailed
```

Comment:
The name is conflicting with other alerts. How about something like `KMCFetchingClustersFromKEBFailed` or similar?

Reply:
Done